test/e2e/run.sh

```diff
@@ -2,7 +2,7 @@
 
 cd "$(dirname "$0")"
 
-## start the php server 
+## start the php server
 # php -S localhost:8888 -t ../../ &
 # SERVER_PID=$!
 
```

Comment:
I think this file is no longer necessary:
- developer starts server with vagrant and use npm scripts to run tests
- travis starts server and run tests according to its configuration

Reply:
Yep, you are right. I will remove it